### PR TITLE
PXC-3899 Release notes and release note tasks for 8.0.27-17 (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
added source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst
modified source/conf.py
modified source/release-notes/release-notes_index.rst
removed source/release-notes/Percona-XtraDB-Cluster-5.7.36-31.55.rst

### CODE REVIEW

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+=======================
+
+* :jirabug:`PXC-3729`: Allowed certified high-priority transactions to proceed without lock conflicts.
+* :jirabug:`PXC-3766`: Stopped every XtraBackup-based SST from executing the version-check procedure.
+* :jirabug:`PXC-3704`: The maximum writeset size is based on ``repl.max_ws_size`` when both ``repl.max_ws_size`` and ``wsrep_max_ws_size`` are passed during startup.
+
+Contact Us
+=======================
+
+The `Documentation Contribution Guide <https://github.com/percona/percona-server/blob/8.0/doc/source/contributing.md>`__ describes the methods available to contribute to the Percona Server for MySQL documentation.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst]: Change to https://github.com/percona/percona-xtradb-cluster/blob/8.0/doc/source/contributing.md

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+For free technical help, visit the Percona `Community Forum <https://forums.percona.com/c/mysql-mariadb/percona-server-for-mysql-8-0/20>`__.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst]: Change to https://forums.percona.com/c/mysql-mariadb/percona-xtradb-cluster-8-x/37

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+To report bugs or submit feature requests, open a `JIRA <https://jira.percona.com/projects/PS/issues>`__ ticket.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst]: Change to https://jira.percona.com/projects/PXC/issues

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst]: Updated in Useful-links.txt file

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst]: Removed

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst]: Removed

**Review by ghost — APPROVED** [review]

File: doc/source/_res/useful-links.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Useful Links
+=======================
+
+The following are useful links:
```
Comment (alina-derkach-oaza) [inline doc/source/_res/useful-links.txt]: I would remove this sentence since it duplicates the heading.

**Review by george-lorch — APPROVED** [review]

File: doc/source/_res/useful-links.txt (hunk 4 above)
Comment (patrickbirch) [reply doc/source/_res/useful-links.txt]: Done.

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**